mindmeld/path.py

```diff
@@ -160,10 +160,11 @@ def safe_path(func):
     @wraps(func)
     def _wrapper(*args, **kwargs):
         res = func(*args, **kwargs)
+        # Replace with relpath to avoid replacing : (root dir) with _ in Windows
         if isinstance(res, tuple):
-            return tuple(map(lambda x: x.replace(":", "_") if x else x, res))
+            return tuple(map(lambda x: os.path.relpath(x).replace(":", "_") if x else x, res))
         elif isinstance(res, str):
-            return res.replace(":", "_")
+            return os.path.relpath(res).replace(":", "_")
```

Comment:
Arent we returning relpaths sometimes and abs path on other times? I think we should standardize it, no?